src/libraries/System.Net.WebSockets.Client/ref/System.Net.WebSockets.Client.cs

```diff
@@ -36,7 +36,7 @@ public sealed partial class ClientWebSocketOptions
         [System.Runtime.Versioning.UnsupportedOSPlatformAttribute("browser")]
         public System.TimeSpan KeepAliveInterval { get { throw null; } set { } }
         [System.Runtime.Versioning.UnsupportedOSPlatformAttribute("browser")]
-        public System.Net.WebSockets.WebSocketDeflateOptions? DeflateOptions { get { throw null; } set { } }
+        public System.Net.WebSockets.WebSocketDeflateOptions? DangerousDeflateOptions { get { throw null; } set { } }
```

Comment:
This change would need to go back via api review?

Reply:
See this: https://github.com/dotnet/runtime/issues/31088#issuecomment-810509834